doc/api/child_process.md

```diff
@@ -390,6 +393,7 @@ changes:
     done on Windows. Ignored on Unix. **Default:** `false`.
   * `uid` {number} Sets the user identity of the process (see setuid(2)).
   * `gid` {number} Sets the group identity of the process (see setgid(2)).
+  * `signal` {AbortSignal} allows closing the subprocess using an AbortSignal.
```

Comment:
```suggestion
  * `signal` {AbortSignal} Allows closing the subprocess using an AbortSignal.
```

Reply:
Optional nit: The options are in alphabetical order up until `uid`. Move `uid`, `gid`, and `signal` to the locations in the list to make the list fully alphabetical order.